app/javascript/components/CreateSurvey.js

```diff
@@ -1,24 +1,25 @@
 import React, { useState, useEffect } from 'react';
-import Card from '@mui/material/Card'
-import Button from '@mui/material/Button'
-import TextField from '@mui/material/TextField';
-import FormControl from '@mui/material/FormControl';
-import Select from '@mui/material/Select';
-import MenuItem from '@mui/material/MenuItem';
-import InputLabel from '@mui/material/InputLabel';
-import Box from '@mui/material/Box';
-import Stack from '@mui/material/Stack';
-import Paper from '@mui/material/Paper';
-import DeleteIcon from '@mui/icons-material/Delete';
-import McQuestion from './McQuestion'
+import {
+    Button,
+    TextField,
+    FormControl,
+    Select,
+    MenuItem,
+    InputLabel,
+    Box,
+    Stack,
+    Paper } from '@mui/material'
+
+import TextQuestion from './questionTypes/TextQuestion';
+import MultipleChoice from './questionTypes/MultipleChoice';
 
 const questionType = {
     "MULTIPLE_CHOICE": "multipleChoice",
     "OPEN_ENDED": "openEnded",
     "NUMERICAL": "numerical",
```

Comment:
remove numerical from the dropdown because we don't have the backend or frontend to support it

Reply:
^ can you delete this line as well @JudyHamwi then you're good to merge